File: BubbleSort.java
import java.util.Arrays;
import java.util.logging.Logger;

public class BubbleSort {
    static Logger logger = Logger.getLogger("");

    class Sort {
        public static void bubbleSort(int[] haystack) {
            int n = haystack.length;
            boolean swapped;
            int temp;
            for (int i = 0; i < n; i++) {
                swapped = false;
                for (int j = 0; j < n - i - 1; j++) {
                    if (haystack[j] > haystack[j + 1]) {
                        temp = haystack[j];
                        haystack[j] = haystack[j + 1];
                        haystack[j + 1] = temp;
                        swapped = true;
                    }
                }
                if (swapped = false) {
                    break;
                }
            }
        }
    }

    public static void main(String[] args) {
        int[] unsorted = new int[] { 9, 1, 5, 43, 32, 22, 19, 90, 55, 66, 45, 88 };
        Sort.bubbleSort(unsorted);
        logger.info(Arrays.toString(unsorted));
    }
}

File: BubbleSort.py
def bubbleSort(array):
    n = len(array)
    swapped = False
    for i in range(n):
        for j in range(n - i - 1):
            if array[j] > array[j + 1]:
                array[j], array[j + 1] = array[j + 1], array[j]
                swapped = True
        if swapped == False:
            break

arr = [9, 1, 5, 43, 32, 22, 19, 90, 55, 66, 45, 88]
bubbleSort(arr)
print(arr)